### PR TITLE
[ES] content/es/mutual-transport-layer-security.md

### LINKED ISSUE #1397: [Spanish] localize 'mTLS (Mutual Transport Layer Security)'
Localize the `content/en/mutual-transport-layer-security.md` document to Spanish

### PR DESCRIPTION
**Pull request by marianogg9**
### Describe your changes
- added `es/mutual-transport-layer-security.md`.

### Related issue number or link (ex: `resolves #issue-number`)
- resolves #1397 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by electrocucaracha — APPROVED** [review]
/lgtm

File: content/es/mutual-transport-layer-security.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Seguridad mutua de capa de transporte (mTLS)
+status: Completed
+category: Concepto
+tags: ["security", "networking", ""]
+---
+
+## ¿Qué es?
+
+La seguridad mutua de capa de transporte (mTLS en Inglés) es una técnica utilizada para autenticar y codificar mensajes enviados entre dos [servicios](/services).
```
Comment (ramrodo) [inline content/es/mutual-transport-layer-security.md:10]: La URL correcta es [/service](https://github.com/cncf/glossary/blob/main/content/en/mutual-transport-layer-security.md?plain=1#L10)

```suggestion
La seguridad mutua de capa de transporte (mTLS en Inglés) es una técnica utilizada para autenticar y codificar mensajes enviados entre dos [servicios](/service).
```

File: content/es/mutual-transport-layer-security.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Los [microservicios](/es/microservices-architecture/) se comunican a través de una red y,
+como las redes de wifi, la comunicación en tránsito en la red puede ser atacada.
+mTLS asegura que ningún agente no autorizado pueda escuchar o personificar peticiones legítimas.
+
+## ¿Cómo ayuda?
+
+mTLS asegura que el tráfico es seguro y confiable en ambas direcciones entre un cliente y un servidor,
+proveyendo una capa adicional de seguridad para los usuarios que utilizan una red o aplicación.
+También verifica las conexiones con dispositivos que no tienen un proceso de autenticación, como un dispositivo de Internet de las Cosas.
+Los ataques tales como personificación, robo de credenciales, fuerza bruta, etc, pueden ser prevenidos por mTLS.
```
Comment (ramrodo) [inline content/es/mutual-transport-layer-security.md:25]:
```suggestion
Los ataques tales como personificación, robo de credenciales, fuerza bruta, etc., pueden ser prevenidos por mTLS.
```

**Review by ramrodo — CHANGES_REQUESTED** [review]
Un par de comentarios :)

**Review by ramrodo — APPROVED** [review]
LGTM

**Pull request merged**